### PR TITLE
[GradleV2] Add spotbugs plugin inputs to Gradle task

### PR DESCRIPTION
**Pull request by vladislav-ryzhov**
**Task name**: GradleV2

**Description**: Added spotbugsTool instance to gradletask.ts. Added inputs for spotbugs plugin. Changed help markdown for findbugs plugin. 

**Documentation changes required:** Y

**Added unit tests:** (Y/N) N

**Attached related issue:** (Y/N) https://github.com/microsoft/azure-pipelines-tasks/issues/13980

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/GradleV2/task.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 2,
-        "Minor": 181,
+        "Minor": 185,
```
Comment (anatolybolshakov) [inline Tasks/GradleV2/task.json:19]: Should there be update for task.loc.json file also?

File: Tasks/GradleV2/task.json [hunk 2]
```diff
@@ -322,6 +322,45 @@
...
+            ],
+            "type": "boolean",
+            "label": "Run SpotBugs",
+            "required": true,
+            "defaultValue": "false",
+            "groupName": "CodeAnalysis",
+            "helpMarkDown": "Enable this option to run spotBugs. This plugin works with Gradle v5.6 or later. [More info](https://spotbugs.readthedocs.io/en/stable/gradle.html#use-spotbugs-gradle-plugin)"
+        },
+        {
+            "name": "spotBugsGradlePluginVersionChoice",
```
Comment (EzzhevNikita) [inline Tasks/GradleV2/task.json:339]: I don't see where this input is used in your code. What it was added for?
The same question for the _spotbugsGradlePluginVersion_ input.

Comment (vladislav-ryzhov) [reply Tasks/GradleV2/task.json:339]: @EzzhevNikita 
You can see it in this spotbugsTool file in this PR https://github.com/microsoft/azure-pipelines-tasks/pull/14618/files
Those inputs are used in spotbugsTool. 
I implemented the spotbugs plugin in the same way as the sonarqube plugin.
image
image

Comment (EzzhevNikita) [reply Tasks/GradleV2/task.json:339]: Got it, thank you for the explanation

File: Tasks/GradleV2/task.json (hunk 1 above)
Comment (vladislav-ryzhov) [reply Tasks/GradleV2/task.json:19]: @anatolybolshakov Yes, updated.

File: Tasks/GradleV2/Strings/resources.resjson/en-US/resources.resjson [hunk 3]
```diff
@@ -47,9 +47,15 @@
...
   "loc.input.label.findbugsAnalysisEnabled": "Run FindBugs",
-  "loc.input.help.findbugsAnalysisEnabled": "Use the FindBugs static analysis tool to look for bugs in the code. Results are uploaded as build artifacts.",
+  "loc.input.help.findbugsAnalysisEnabled": "Use the FindBugs static analysis tool to look for bugs in the code. Results are uploaded as build artifacts. In Gradle 6.0 this plugin was removed. Use spotbugs plugin instead. [More info](https://docs.gradle.org/current/userguide/upgrading_version_5.html#the_findbugs_plugin_has_been_removed)",
   "loc.input.label.pmdAnalysisEnabled": "Run PMD",
   "loc.input.help.pmdAnalysisEnabled": "Use the PMD Java static analysis tool to look for bugs in the code. Results are uploaded as build artifacts.",
+  "loc.input.label.spotBugsAnalysisEnabled": "Run SpotBugs",
+  "loc.input.help.spotBugsAnalysisEnabled": "Enable this option to run spotBugs. This plugin works with Gradle v5.6 or later. [More info](https://spotbugs.readthedocs.io/en/stable/gradle.html#use-spotbugs-gradle-plugin)",
+  "loc.input.label.spotBugsGradlePluginVersionChoice": "Spotbugs plugin for Gradle version",
```
Comment (anatolybolshakov) [inline Tasks/GradleV2/Strings/resources.resjson/en-US/resources.resjson:55]: This sounds little bit confusing (plugin or gradle version?)

**Review by anatolybolshakov — APPROVED** [review]
LGTM, thanks! Could you please take a look at the comments?
It would be great also to test it once related changes for common package will be merged.

File: Tasks/GradleV2/Strings/resources.resjson/en-US/resources.resjson (hunk 3 above)
Comment (vladislav-ryzhov) [reply Tasks/GradleV2/Strings/resources.resjson/en-US/resources.resjson]: @anatolybolshakov Okay, corrected it.

**Review by max-zaytsev — APPROVED** [review]

File: Tasks/GradleV2/Strings/resources.resjson/en-US/resources.resjson (hunk 3 above)
Comment (max-zaytsev) [reply Tasks/GradleV2/Strings/resources.resjson/en-US/resources.resjson]: It's still confusing :)

image

**Pull request merged**